octopoes/octopoes/models/transaction.py

```diff
@@ -0,0 +1,12 @@
+from datetime import datetime
+from typing import Dict, Optional
+
+from pydantic import BaseModel
+
+
+class TransactionRecord(BaseModel):
+    txTime: datetime
+    txId: int
+    validTime: datetime
+    contentHash: str
+    doc: Optional[Dict]
```

Comment:
I strongly suggest to use [field aliases](https://docs.pydantic.dev/1.10/usage/schema/#field-customization). This way you can still support external data structures while ensuring Python naming conventions

Reply:
Doc [link](https://docs.pydantic.dev/2.5/concepts/alias/) for Pydantic v2